spec/domain/app_status/mail_spec.rb

```diff
@@ -79,6 +80,30 @@
 
   end
 
+  context 'truemail' do
+    before do
+      cache.write(:app_status, { seen_mails: seen_mails})
+      expect(imap_connector)
+        .to receive(:fetch_mails)
+        .with(:inbox)
+        .and_return([])
+    end
+
+    it "doesn't verify correctly anymore" do
+      allow(Truemail).to receive(:valid?).and_return(false)
+
+      expect(app_status.code).to eq(:ok)
```

Comment:
should this still have status code :ok if true mail failing ?

Reply:
I wasn't sure about that either. In theory, the mail server is still working and only the saving of a person is impaired, therefore the :service_unavailable status would be wrong. But we might could implement another fitting status if only Truemail would fail. Do you have any suggestions for a good matching status?